app/models/response_choice.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: response_choices
+#
+#  id                                   :integer          not null, primary key
+#  question_id                          :integer
+#  key                                  :string
+#  text                                 :text
+#  created_at                           :datetime         not null
+#  updated_at                           :datetime         not null
+#
+
+class ResponseChoice < ActiveRecord::Base
+  belongs_to :question
+  has_many :question_orders
+  validates :key, presence: true
```

Comment:
What are keys suppose to represent?

For MC, i.e. for a question asking about favourite city - would key be `a)` and text `Toronto`.

If that's the case what would a key be for short answers? I suppose it can have a special key like `short_answer`